src/containers/TurnitinDisplay/index.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import {
+  Card, Collapsible, Icon, DataTable, Button,
+} from '@edx/paragon';
+import { ArrowDropDown, ArrowDropUp, WarningFilled } from '@edx/paragon/icons';
+import messages from './components/messages';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import FileNameCell from './components/FileNameCell';
+import ButtonCell from './components/ButtonCell';
+import createDOMPurify from 'dompurify';
+
+
+/**
+ * <TurnitinDisplay />
+ */
+export class TurnitinDisplay extends React.Component {
```

Comment:
I recommend to use functional component instead of class component

Reply:
Thank you! I just updated the PR